package/python-kiwi-spec-template

```diff
@@ -746,6 +746,16 @@ fi
 %endif
 %endif
 
+%post -n dracut-kiwi-oem-repart
+#============================================================
+# create /etc/dracut.conf.d/40-kiwi-resize.conf
+mkdir -p /etc/dracut.conf.d
+echo 'add_dracutmodules+=" kiwi-repart "' > /etc/dracut.conf.d/40-kiwi-resize.conf
```

Comment:
I wonder what happens if someone installs this RPM in a host, won't this result into a generating a new initrd including the resize module? Probably even worse because the module would be only added to initrd if it is regenarated which does not happen automatically by installing this package.

The fact that installing this RPM in a regular host day A adding a dracut setup that on day B (e.g. after a kernel update) will cause resize module to be included is a bit scary to my eyes. In most cases this should not really apply any change, but on certain cases I fear it could silently re-partition a host...

IIRC the reason we run `check() { return 255 }` in our modules is to prevent auto inclusion of such a module in initrd. So they are only included into the initrd if it is explicitly required.

Reply:
@davidcassany I understand, how about changing the %post to add this dracut conf if it can find `/config.partids` in the system ? This clearly states it's a kiwi image that got deployed and it has the resize activated in the original image and has also successfully booted into the system with that resize code active

Reply:
I pushed a commit to that idea